components/gitpod-cli/cmd/rebuild.go

```diff
@@ -222,9 +222,7 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 		return err
 	}
 
-	// TODO project? - should not it be covered by gp env?
```

Comment:
we fixed it on Monday, now `gp env` actually return all workspace env vars, i.e. from user, project and context